custom_components/localtuya/cover.py

```diff
@@ -84,15 +84,15 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     #_LOGGER.info("conf_STOP_cmd is %s", config.get(CONF_STOP_CMD))
 
     covers = []
-    pytuyadevice = pytuya.CoverEntity(config.get(CONF_DEVICE_ID), config.get(CONF_HOST), config.get(CONF_LOCAL_KEY))
+    pytuyadevice = pytuya.PytuyaDevice(config.get(CONF_DEVICE_ID), config.get(CONF_HOST), config.get(CONF_LOCAL_KEY))
```

Comment:
It seems a bit redundant to call it `PytuyaDevice`, can't we just stick to `TuyaDevice`?

Reply:
Agree, done